pkg/bgpv1/types/bgp.go

```diff
@@ -104,3 +105,8 @@ type Router interface {
 	// GetBGP returns configured BGP global parameters
 	GetBGP(ctx context.Context) (GetBGPResponse, error)
 }
+
+// PtrPort returns the provided port as a pointer.
+func PtrPort(port int) *int {
+	return &port
+}
```

Comment:
Moved this func to the types pkg since it's more intuitive for imports than the agent pkg.